### ISSUE TITLE
Current release doesn't build

### ISSUE THREAD

**Issue by svenstaro** [luvit/luv#376]
The last release file luv-1.30.1-0.tar.gz doesn't build because it doesn't include the pkg-config file `libluv.pc.in` so CMake complains:

```
CMake Error: File /build/libluv/src/luv-1.30.1-0/libluv.pc.in does not exist.
CMake Error at CMakeLists.txt:206 (configure_file):
  configure_file Problem configuring file


-- Configuring incomplete, errors occurred!
```
A new release with that file included should be made.

**Comment by squeek502** [comment]
Agreed, will release a `1.30.1-1` when I get a chance.

**Comment by squeek502** [comment]
Should be fixed in `1.30.1-1`.
